course-material/advanced-querying-techniques/subqueries.md

```diff
@@ -1,14 +1,497 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: jupyblog
+  language: python
+  name: python3
 ---
 
 # Using subqueries
 
+A subquery is a technique that essentially embeds a query within another query. It enables users to run a query that has some kind of relation to another query in the same SQL statement. 
 
+Let's demonstrate how powerful subqueries are by continuing with our banking data.
 
+<!-- region -->
+
+## Install - execute this once.
+
+```{important}
+<b>Note:</b> If you are following these lessons locally and <b>not</b> on Google Colab, then there is no need to reinstall these packages.
```

Comment:
Change the note to say:

"The `--save` and `%sqlcmd` features used required the latest JupySQL version. Ensure you run the code below"

Reply:
The note has been udpated.